cdk-infra/lib/constructs/api/webhook-api-construct.ts

```diff
@@ -74,6 +74,15 @@ export class WebhookApiConstruct extends Construct {
       timeout,
     });
 
+    const githubDeleteArtifactsLambda = new NodejsFunction(this, 'githubDeleteArtifactsLambda', {
+      entry: `${HANDLERS_PATH}/github.ts`,
+      runtime,
```

Comment:
That makes sense, and I agree. I think we should make it a separate task to avoid the risk of introducing any potential regression so soon to UAT. I can create a ticket for something like this later